exchange/utils_test.go

```diff
@@ -2734,47 +2623,6 @@ func TestBuildXPrebidHeader(t *testing.T) {
 	}
 }
 
-func TestSourceExtSChainCopied(t *testing.T) {
```

Comment:
This test was deleted because the corresponding logic was moved into the schain writer implementation `Write` methods which have tests that cover this logic.